packages/terra-framework-docs/src/terra-dev-site/doc/collapsible-menu-view/example/AlwaysCollapsedMenuItemsDemo.jsx

```diff
@@ -66,6 +67,7 @@ class AlwaysCollapsedMenuItemsDemo extends React.Component {
             icon={<IconVisualization />}
             text="Chart View"
             key="chartView"
+            ariaDescribedBy="view-type-single-select"
```

Comment:
Can we add this attribute to menu-item instead adding them on the examples. to ensure better A11y experience